src/net.c

```diff
@@ -1368,6 +1464,104 @@ coap_read(coap_context_t *ctx, coap_tick_t now) {
   }
 }
 
+#ifdef COAP_EPOLL_SUPPORT
+/*
+ * While this code in part replicates coap_read(), doing the functions
+ * directly saves having to iterate through the endpoints / sessions.
+ */
+int
+coap_io_do_events(coap_context_t *ctx, struct epoll_event *events, int nevents) {
```

Comment:
As `nevents` is for counting array elements, I suggest `size_t` as data type.

Reply:
Fixed